webpack/assets/javascripts/react_app/components/common/forms/RadioButtonGroup.test.js

```diff
@@ -0,0 +1,21 @@
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import RadioButtonGroup from './RadioButtonGroup';
+
+describe('radio button group', () => {
+  const radios = [
+    {
+      label: 'A', checked: true, value: 'A',
+    },
+    {
+      label: 'B', checked: false, value: 'B',
+    },
+  ];
+
+  it('should render group of radio buttons', () => {
+    const wrapper = shallow(<RadioButtonGroup name="RadioButtonGroupTest" controlLabel="RadioButtonGroupLabel" radios={radios} />);
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
```

Comment:
could you use `Prettier` format ?
for example it would look like:
```javascript
    const wrapper = shallow(
      <RadioButtonGroup
        name="RadioButtonGroupTest"
        controlLabel="RadioButtonGroupLabel"
        radios={radios}
      />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
```

Reply:
Running Prettier out of the box gives me incompatibilities with our linting rules. Formatting the code in test manually according to your example gives me still linting offences, namely:

![](https://i.imgur.com/QLTkYSI.png)

I am very confused. Did we decide to use Prettier as a style guide? If so, then why don't we have a [pinned version](https://prettier.io/docs/en/install.html)  in `package.json` and CI job that would check and enforce the style?

If not, then why require ad-hoc code styling? Don't we have linting for this?

Reply:
I know that there were some attemptes to combine lint and Prettier with no success.
the best practice for now is to run Prettier first and lint after.

Reply:
`npm run lint -- --fix` usually fix the errors from prettier